tsdb/index/tsi1/log_file.go

```diff
@@ -544,21 +544,22 @@ func (f *LogFile) execSeriesEntry(e *LogEntry) {
 	// Generate key & series, if not exists.
 	// TODO(edd) use a pool of buffers?
 	key := AppendSeriesKey(nil, e.Name, e.Tags)
-	serie := mm.series[string(key)]
+	keyStr := string(key)
+	serie := mm.series[keyStr]
```

Comment:
Did this actually reduce allocations?

https://github.com/golang/go/wiki/CompilerOptimizations#map-lookup-by-byte

> For a map m of type `map[string]T` and `[]byte b`, `m[string(b)]` doesn't allocate. (the temporary string copy of the byte slice isn't made)